### DISCUSSION TITLE
Official repo just added new web interface -- will these be included?

### DISCUSSION THREAD

**Discussion by argumemnon** [haugene/docker-transmission-openvpn#2376] [Q&A]
Hello, the official transmission repo (transmission/transmission) just added a [refreshed web interface across desktop and mobile devices](https://github.com/transmission/transmission/commit/6187cfd67b269bf29644bc068bb840da980f718b). Will we see these changes pulled into this docker implementation? 

I have searched around and found [this discussion](https://github.com/haugene/docker-transmission-openvpn/discussions/1937) but I am not totally clear as to what it all means. I assume that this repo won't be pulling in those changes for a while since "We've made some choices to go more in the LTS direction instead of bleeding edge. We will update packages slowly and steadily but will not prioritize to be on any bleeding edge."

**Comment by ilike2burnthing** [comment]
Would possibly be added in the future - https://github.com/haugene/docker-transmission-openvpn/discussions/2357
